content/02.main-text.md

```diff
@@ -108,10 +108,30 @@ The notebook to generate this figure can be [interactively launched using Binder
 
 ## Manubot features
 
-Manubot is a system for writing scholarly manuscripts via GitHub that is built upon our [Python package](https://github.com/manubot/manubot) of the same name.
-With Manubot, manuscripts are written as plain-text Markdown files, which is well suited for version control using git.
-The Markdown standard itself provides limited yet crucial formatting syntax, including the ability to embed images and format text via bold, italics, hyperlinks, headers, inline code, codeblocks, blockquotes, and numbered or bulleted lists.
+Manubot is a system for writing scholarly manuscripts via GitHub.
+For each manuscript, there is a corresponding Git repository.
+The `master` branch of the repository contains all of the necessary inputs to build the manuscript.
+Specifically, a `content` directory contains one-or-more Markdown files that define the body of the manuscript as well as a YAML-formatted file specifying manuscript metadata (e.g. the title, authors, keywords, and language).
+Figures can be hosted in the `content/images` subdirectory or elsewhere and specified by URL.
+Repositories contain scripts and other assets that define how to build and deploy the manuscript.
+Many of these operations are delegated to the `manubot` [Python package](https://github.com/manubot/manubot) or other dependencies such as Pandoc and Travis CI.
+Manubot pieces together many existing standards and technologies to enable encapsulating a manuscript in a repository where the generation of outputs is automated.
```

Comment:
I imagine this paragraph may need be rewritten to be more accessible. However, I thought it was necessary to fill in several missing details of the current infrastructure. Feedback please!